tests/ui/perfherder/alerts_test.jsx

```diff
@@ -202,6 +203,43 @@ test('toggle buttons should filter alert summary and alerts by selected filter',
   expect(alert2).not.toBeInTheDocument();
 });
 
+test('alert filtering ignores repository and options, but filters by platform', async () => {
```

Comment:
This test isn't entirely precise. First of all, there's a distinction between `options` and `extra_options`.
Perfherder is able to filter by `extra_options` on both server & client side. The `options` cannot be filtered on server side; thus, when filtering by `options` in list mode, we aren't able to see anything.

This test's description is ambiguous when referring to options & contradicts its checks, as it actually considers the `e10s` extra_option.

Reply:
Moved into `testAlertSummaries[0].alerts[0].series_signature.options`